extensions/2.0/Khronos/KHR_xmp/README.md

```diff
@@ -0,0 +1,336 @@
+# KHR_xmp
+
+## Contributors
+
+* Emiliano Gambaretto, Adobe, <mailto:emiliano@adobe.com>
+* Mike Bond, Adobe, [@MiiBond](https://twitter.com/MiiBond)
+* Robert Long, Mozilla, [@arobertlong](https://twitter.com/arobertlong)
+* Don McCurdy, Google, [@donmccurdy](https://twitter.com/donrmccurdy)
+* Andreas Plesch, Individual Contributor,[@andreasplesch](https://github.com/andreasplesch)
+* Alexey Knyazev, Individual Contributor, [@lexaknyazev](https://github.com/lexaknyazev)
+* Ed Mackey, Analytical Graphics, Inc.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds support for [XMP (Extensible Metadata Platform)](https://github.com/adobe/xmp-docs) metadata to glTF.
+XMP is a technology for embedding metadata into documents and has been [an ISO standard since 2012](https://www.iso.org/news/2012/03/Ref1525.html).
+XMP metadata is embedded in a top-level glTF extension as an array of metadata packets.
+XMP metadata packets can then be referenced from glTF objects of type: `asset`, `scene`, `node`, `mesh`, `material`, `image`, `animation`.
+XMP metadata referenced by the glTF top level object `asset` applies to the entire glTF asset.
+XMP metadata is organized in namespaces. This extension enables any XMP metadata namespace to be embedded in a glTF asset.
+We refer to the [XMP namespaces documentation](https://github.com/adobe/xmp-docs/tree/master/XMPNamespaces) for a detailed description of XMP properties.
+
+## XMP data types
+
+This section describes how [XMP data types](https://github.com/adobe/xmp-docs/tree/master/XMPNamespaces/XMPDataTypes) shall be encoded in a glTF JSON.
+XMP supports three classes of data types: simple, structure and array. XMP structures are encoded via JSON objects. XMP arrays are encoded via JSON arrays.
```

Comment:
Note that there are no XMP arrays present in the core properties below, and therefore no JSON arrays defined by this extension. Is this correct? Surely some XMP namespaces use arrays? I may be missing something.

Reply:
Never mind, I think I see. Data types (e.g. arrays) may contain objects of the "core properties" types below, but are not themselves core properties.